microverse_collins_yemi-main/fcc-test-suite-template-master/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div class="main">
+    <div>
+      <h1>Handcrafted, home-made masterpieces</h1>
+      <input placeholder="Enter your email">
+      <h1 class="bgYellow">SELECT</h1>
+    </div>
+
+    <div>
+      <h1>Premium Materials</h1>
+      <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
+    </div>
```

Comment:
Please I think replacing the div tag with the section tag will make it semantically correct and you have also repeated the h1 mistake again, try to use other hx tags.

```suggestion
    <section>
      <h1>Premium Materials</h1>  
      <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
    </section>
```